doc/operator-creation.md

```diff
@@ -98,9 +97,49 @@ Observable.prototype.mySimpleOperator = mySimpleOperator;
 someObservable.mySimpleOperator(x => x + '!');
 ```
 
+### Operator as a pure function
+
+If you don't want to patch the Observable prototype, you can also write the operator as a pure function that takes the input Observable as argument, instead of relying on the `this` keyword.
+
+Example implementation:
+
+```js
+function mySimpleOperator(someCallback) {
+  // notice that we return a function here
+  return function mySimpleOperatorImplementation(source) {
+    return Observable.create(subscriber => {
+      var subscription = source.subscribe(value => {
+        try {
+          subscriber.next(someCallback(value));
```

Comment:
Yeah I would have to change the previous example too. I can do that, but just saying